### PR TITLE
Extension field batch inverse + minor cleanup of barycentric eval kernel bindings

### PR DESCRIPTION
**Pull request by mcarilli**
Companion to https://github.com/matter-labs/era-shivini/pull/5

### CODE REVIEW

File: boojum-cuda/native/barycentric.cu [hunk 1]
```diff
@@ -162,4 +162,4 @@ EXTERN __launch_bounds__(1024, 1) __global__ void batch_barycentric_partial_redu
   batch_barycentric_partial_reduce_impl<extension_field>(batch_ys, lagrange_coeffs, partial_sums, log_count, num_polys);
 }
 
-} // namespace goldilocks
\ No newline at end of file
+} // namespace goldilocks
```
Comment (robik75) [inline boojum-cuda/native/barycentric.cu:165]: This looks like some unrelated formatting change.

File: boojum-cuda/native/ops_complex.cu [hunk 2]
```diff
@@ -165,8 +165,17 @@ EXTERN __global__ void select_kernel(const unsigned *indexes, const base_field *
...
+template <typename T> struct InvBatch {};
+template <> struct InvBatch<base_field> {
+  static constexpr unsigned INV_BATCH = 10;
+};
+template <> struct InvBatch<extension_field> {
+  static constexpr unsigned INV_BATCH = 6;
+};
+
+template<typename T, typename SETTER, typename GETTER>
+DEVICE_FORCEINLINE void batch_inv_impl(SETTER src, GETTER dst, const unsigned count) {
```
Comment (robik75) [inline boojum-cuda/native/ops_complex.cu:177]: Is the SETTER-GETTER meaning reversed here?

Comment (mcarilli) [reply boojum-cuda/native/ops_complex.cu:177]: oops, yes

File: boojum-cuda/native/barycentric.cu (hunk 1 above)
Comment (mcarilli) [reply boojum-cuda/native/barycentric.cu]: fixed

**Comment by robik75** [comment]
replaced by #9
